### DISCUSSION TITLE
What should be in the v0.2.0 milestone?

### DISCUSSION THREAD

**Discussion by bryanrcarlson** [CafIncubator/Midden#67] [Ideas]
Hi Middenites (Midden-heads)?

Any thoughts or strong feelings about what features we should focus on next? Here are some ideas:

- **Projects**: include a "project" tab in the Catalog that lists all projects within an organization, including project descriptions. Clicking a project will list all datasets belonging to that project. 
- **Usability**: things like validation in the Editor, maybe auto-generating variables from a sample data file (e.g. upload a csv and extract headings), local caching of edits so prevent loss of work when browser crashes. 
- **Interoperability**: things like exporting .middens to different metadata formats; integration within LTAR Inventory; export variables as a csv (data dictionary)
- **Focus on tools for downloading actual data**: the MiddenCli has read and list access to data stores should could grab datasets too; the details get complicated though.

Or something else entirely? What does your group need most?

Personally, my group needs better Project support at the moment. But figuring out how use the MiddenCli to download datasets from the Catalog would be awesome too.

**Comment by riemino** [comment]
Relating to your interoperability idea, it would be cool if we could also go from the data inventory to midden files.  This way, we would populate the site's Midden installation using the work already done in the data inventory. Just $0.02
